src/main/resources/public/jsx/DatasetForm.jsx

```diff
@@ -22,8 +22,7 @@ class DatasetForm extends React.Component {
         organizations:React.PropTypes.array.isRequired,
         datasetName: React.PropTypes.string.isRequired,
         onChangeNotif: React.PropTypes.func.isRequired,
-        geoLocation: React.PropTypes.object.isRequired,
-        private : React.PropTypes.bool.isRequired
```

Comment:
does not seem normal ?

Reply:
I just removed private property that was not mandatory.
geoLocation is kept.

Reply:
Rollback : private property is set to required
`private` is provided with default value at DatasetForm instantiation (see last commit)